src/vs/editor/standalone/browser/standaloneServices.ts

```diff
@@ -226,13 +232,20 @@ export class DynamicStandaloneServices extends Disposable {
 
 		let keybindingService = ensure(IKeybindingService, () => this._register(new StandaloneKeybindingService(contextKeyService, commandService, telemetryService, notificationService, logService, domElement)));
 
-		let layoutService = ensure(ILayoutService, () => new SimpleLayoutService(StaticServices.codeEditorService.get(ICodeEditorService), domElement));
+		let layoutService = ensure(ILayoutService, () => new SimpleLayoutService(codeEditorService, domElement));
 
-		ensure(IQuickInputService, () => new StandaloneQuickInputServiceImpl(_instantiationService, StaticServices.codeEditorService.get(ICodeEditorService)));
+		ensure(IQuickInputService, () => new StandaloneQuickInputServiceImpl(_instantiationService, codeEditorService));
 
 		let contextViewService = ensure(IContextViewService, () => this._register(new ContextViewService(layoutService)));
 
-		ensure(IClipboardService, () => new BrowserClipboardService());
+		const openerService = ensure(IOpenerService, () => new OpenerService(codeEditorService, commandService));
+
+		ensure(IClipboardService, () => new BrowserClipboardService(
+			dialogService,
+			openerService,
+			logService,
+			this.get(IEnvironmentService) // TODO: this will throw
```

Comment:
I left a TODO behind, `IEnvironmentService` is not available in the standalone editor, so this will throw.